fbw-a32nx/src/systems/fmgc/src/flightplanning/FlightPlanService.ts

```diff
@@ -17,24 +16,29 @@ import { FlightPlanInterface } from '@fmgc/flightplanning/FlightPlanInterface';
 import { AltitudeConstraint } from '@fmgc/flightplanning/data/constraint';
 import { CopyOptions } from '@fmgc/flightplanning/plans/CloningOptions';
 import { FlightPlanPerformanceData } from '@fmgc/flightplanning/plans/performance/FlightPlanPerformanceData';
+import { FlightPlanBatch } from '@fmgc/flightplanning/plans/FlightPlanBatch';
 
 export class FlightPlanService<P extends FlightPlanPerformanceData = FlightPlanPerformanceData>
   implements FlightPlanInterface<P>
 {
   private readonly flightPlanManager: FlightPlanManager<P>;
 
-  navigationDatabase: NavigationDatabase;
+  public syncClientID = Math.round(Math.random() * 10_000_000);
```

Comment:
While incredibly unlikely, multiple instances could potentially get the same `syncClientID` here. Could this lead to issues?

Reply:
It would cause issues yes. I can bump up the constant for now - could be a UUID, but I feel that allocation would be more expensive when deserializing sync messages.

Reply:
Made the RNG better.